build.sh

```diff
@@ -0,0 +1,6 @@
+curl -L https://github.com/alphagov/govuk-frontend/releases/download/v4.0.0/release-v4.0.0.zip > govuk_frontend.zip
```

Comment:
Refactor away from bash - some of our devs will be working on windows machines.  Potentially fold this into the build.py?